indico/modules/categories/controllers/management.py

```diff
@@ -117,6 +122,33 @@ def _process(self):
                                                     icon_form=icon_form, logo_form=logo_form)
 
 
+class RHAPIEventMoveRequests(RHManageCategoryBase):
+    def _process_GET(self):
+        return EventMoveRequestSchema(many=True).jsonify(self.category.event_move_requests)
+
+    @use_kwargs({
+        'state': EnumField(MoveRequestState, required=True)
+    })
+    def _process_POST(self, state):
+        move_requests = parser.parse({
+            'requests': EventRequestList(required=True, category=self.category, validate=not_empty)
+        }, unknown=EXCLUDE)['requests']
+
+        if state not in (MoveRequestState.accepted, MoveRequestState.rejected):
+            raise Forbidden
```

Comment:
(if you end up keeping this: `BadRequest` or `UnprocessableEntity` are probably better - or just do it via `validate=validate_with_message(...)` in the webargs field)